### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sguinetti**
This PR removes links to apps that do not exist, a update link for one app and a new app is added.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,15 +84,13 @@ Applications with support of IPTV streams.
 - [TVirl](https://play.google.com/store/apps/details?id=by.stari4ek.tvirl) - Special Android TV Input Service to integrate IPTV channels to pre-installed system TV App like [Live Channels](https://play.google.com/store/apps/details?id=com.google.android.tv).
 - [CosmiTV Player](https://play.google.com/store/apps/details?id=com.cosmiquest.tv) - IPTV Player and DVR for Android that resembles the cable tv box.
 - [kantv](https://github.com/zhouwg/kantv) - Open source player for Android.
+- [M3UAndroid](https://github.com/realOxy/M3UAndroid) - Open source stream player for Android. This application allows you to play in zapping mode, mark channels as favorites and other media player enhancements.
```
Comment (freearhey) [inline README.md:87]: - new links should be added to the end of the list
- the description should consist of one sentence

File: README.md [hunk 2]
```diff
@@ -85,15 +84,13 @@ Applications with support of IPTV streams.
...
 - [IPTV Smarters App](https://www.iptvsmarters.com/#downloads) - Video streaming player that allows your IPTV customers or end-users to stream content like Live TV, VOD, Series, and TV Catchup supplied by you.
 - [IPTV Smart Purple Player](https://www.purplesmarttv.com/#downloads) - IPTV Player for users to watch content with their playlist.
 - [Mbogi Music](https://play.google.com/store/apps/details?id=com.mbogimusic) - A one stop audio and video app that gives you access to thousands of free online radio, free IPTV and podcasts from all over the world.
-- [Opus IPTV Player](https://play.google.com/store/apps/details?id=com.handy.opus.iptv.xtream.player&referrer=utm_source%3Dawesome-iptv%26utm_medium%3Dlink) - Cross-platform IPTV Player for all your devices including iOS, iPad, MacOS, Apple TV, Android, and Android Tv.
+- [Opus IPTV Player](https://opusmediaplayer.com/pages/transform-android-tv-into-iptv-powerhouse-opus-iptv-player/) - Cross-platform IPTV Player for all your devices including iOS, iPad, MacOS, Apple TV, Android, and Android Tv.
```
Comment (freearhey) [inline README.md:93]: The link can be completely removed since the Android app is no longer available for download.

**Review by freearhey — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (sguinetti) [reply README.md]: I already updated your link to a new one for Opus. Do you want the name and description to be at the end of the list?

**Review by freearhey — APPROVED** [review]

**Comment by sguinetti** [comment]
@BellezaEmporium Could you please review this application? I've been taking care of proposing TV support to its author. But for now, I'm making a PR for the Android client.

**Comment by BellezaEmporium** [comment]
> @BellezaEmporium Could you please review this application? I've been taking care of proposing TV support to its author. But for now, I'm making a PR for the Android client.

Will take a check.

**Review by BellezaEmporium — APPROVED** [review]
LGTM, quite interesting app. Good luck to the developer.

**Comment by sguinetti** [comment]
> LGTM, quite interesting app. Good luck to the developer.

Thank you for your comment. I'm sure the developer is happy and more people will participate in the IPTV software.

**Pull request merged**